actionview/lib/action_view/component.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module ActionView
+  class Component < ActionView::Base
+    include ActiveModel::Validations
```

Comment:
Do we want this by default?  Validating all components will punish performance of components that have no validations.  If most components require validation, it probably makes sense.  I just don't have a good feeling of the requirement.

Reply:
Maybe one more layer called `class ApplicationComponent < ActionView::Component`?
That will be the place to specify template handler as well as including `ActiveModel::Validations`

Reply:
My gut says it would be best to measure the impact of having validations enabled by default. 

If it's not much overhead, I think this would be a nice default.

Reply:
@skyksandr I 💯agree about having an `ApplicationComponent`. We already follow this pattern internally and have found it useful.